client/extensions/woocommerce/app/products/product-form-variations-modal.js

```diff
@@ -82,7 +89,15 @@ class ProductFormVariationsModal extends React.Component {
 
 				<div className="products__product-form-modal-contents">
 					<h2>{ formattedVariationName( variation ) }</h2>
-					<span className="products__product-form-sku">SKU:</span>
+					<FormFieldSet className="products__product-form-details-basic-sku">
+						<FormLabel htmlFor="sku">{ translate( 'SKU:' ) }</FormLabel>
```

Comment:
Same `htmlFor` comment as #14702, which again also applies to other labels/inputs here